src/funit/asserts/Assert_Real.tmpl

```diff
@@ -176,6 +249,7 @@ module pf_AssertReal_{rank}d
    @overload(AssertGreaterThanOrEqual, no_tol)
 
    @overload(AssertRelativelyEqual, with_tol)
+   @overload(AssertRelMinEqual, with_rel_tol)
```

Comment:
Are you sure that you cannot use the same `with_tol` list in this case?    I've not looked at it closely, but I'd think the overloads are the same for both.

Reply:
I lack the confidence to know. I think because the AssertRelMinEqual requires 4 arguments rather than 3, because it requires the `relative tolerance` and the `minimum tolerance`?

Reply:
Another thought here:    could this maybe be accomplished with an extra optional argument on the original routine?   Not wed to that; just want to be sure we've explored the solution space.

Reply:
I am not beholden to any particular approach. I thought creating a new function would reduce the risk to already implemented functions.

Reply:
I tend to agree.   But there's enough duplicate code to continually tweak my "refactor" instinct ...     

Note:  I like the term "approx" you just introduced in the parser.    It might be good to make the Fortran agree a bit better with the parser name.  There are still a few that don't use the parser.      The purist in me sort of leans towards `@assertApproximatelyEqual` (with the Fortran something like assert_approximately_equal), but I see the appeal of brevity here as well if this is going to be used regularly.       So perhaps allow long and short int the parser `@assertApprox()` and `@assertApproximatelyEqual()` with both expanding tho the unabbreviated snake_case Fortran?

Reply:
I like `@assertApproximatelyEqual` as well, but I believe it hits a name length error, at least the way it is written now.

Reply:
I think what you are saying is that parser could provide aliases that map to the fortran names.?

e.g. `approx` -> `ApproxEqual`

Reply:
Yes - but I've not delved deeply in that layer in some time so did not really think it through.     If it's not trivial then its not worth doing.   I'll try to look to see what is possible in that direction.

Reply:
Regarding name length error - this is in the generated Fortran?     Something exceeds the 63 character limit with this spelled out?

Reply:
Here is an example error when I name things `assertApproximatelyEqual`

```
/media/store/krs/build/pfunit/build/src/funit/asserts/AssertReal_0d.F90:234:87:

  234 |       module procedure assertApproximatelyEqual_Int_0d_DblReal_0d_DblReal_0d_DblReal_0d
      |                                                                                       1
Error: Name at (1) is too long
```

here is the branch with the longer names
https://github.com/kurtsansom/pFUnit/tree/long_name_error

Reply:
Ah right - I forgot about the name mangling.  In theory the suffixes could be shortened a bit, but there really is not a lot of wiggle room by the time you have 4 overload parameters.

But ... I think we could safely the overload to both tolerances having the same `KIND`.    If I understand correctly (in haste) this would also allow the reuse of the overload list, while buying you ~10 more characters before hitting the length limit.